README.md

```diff
@@ -1,32 +1,31 @@
-## Submitting an Idea
+# How to create and publish an article at Rootstrap blog page
 
-For the management of proposals and new ideas for blog posts, we use this repo's [issues](https://github.com/rootstrap/blog/issues). If you have an idea for an article, either you want to write it yourself or not, you can create a new issue describing it and tagging it accordingly depending on which category the article would fit in.
+Publishing an article help us to improve Rootstrap brand with quality content on the web. Also writing about a topic, enhance your knowledge of it. If you want to create and publish an article, please follow these simple steps:
 
-To make this process easier, we provide an issue template where we ask you for some basic information on your proposal: what’s the motivation for writing about the topic, some notes on the idea itself and possible keywords to identify the article with.
+1. If you have an idea, [submit it](https://github.com/rootstrap/blog/blob/master/SUBMIT-AN-IDEA.md) as an [issue](https://github.com/rootstrap/blog/issues). Otherwise, you can choose a submitted iddea from this repository issues and assign it to yourself.
```

Comment:
```suggestion
1. If you have an idea, [submit it](https://github.com/rootstrap/blog/blob/master/SUBMIT-AN-IDEA.md) as an [issue](https://github.com/rootstrap/blog/issues). Otherwise, you can choose a submitted idea from this repository issues and assign it to yourself.
```

Reply:
Also, URL `https://github.com/rootstrap/blog/blob/master/SUBMIT-AN-IDEA.md` gives 404 not found

Reply:
Because that file isn't on master yet

Reply:
oh right, sorry.